steam/ext/csgo/protobufs/ranks.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+
+import betterproto
+
+from ....protobufs.msg import GCProtobufMessage
+from ..enums import EMsg
+
+
+@dataclass(eq=False, repr=False)
+class PlayerRankingInfo(betterproto.Message):
+    account_id: int = betterproto.uint32_field(1)
+    rank_id: int = betterproto.uint32_field(2)
+    wins: int = betterproto.uint32_field(3)
+    rank_change: float = betterproto.float_field(4)
+    rank_type_id: int = betterproto.uint32_field(6)
+    tv_control: int = betterproto.uint32_field(7)
+    rank_window_stats: int = betterproto.uint64_field(8)
+    leaderboard_name: str = betterproto.string_field(9)
+    rank_if_win: int = betterproto.uint32_field(10)
+    rank_if_lose: int = betterproto.uint32_field(11)
+    rank_if_tie: int = betterproto.uint32_field(12)
+    per_map_rank: "list[PerMapRank]" = betterproto.message_field(13, repeated=True)
```

Comment:
```suggestion
    per_map_rank: "list[PerMapRank]" = betterproto.message_field(13)
```
This isn't gonna work with betterproto, repeated isn't a kwarg it takes. If you're gonna use co-pilot to generate these you'll need to take a closer look at the other examples. Ideally if you've never used them before you should be using the protoc CLI (see https://github.com/danielgtaylor/python-betterproto?tab=readme-ov-file#compiling-proto-files)

Reply:
Done.